src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,43 @@
+.menu {
+
+  background-color: $secondary-color;
+  height: 100vh;
+
+  &__icon {
+    @include top-actions;
+    margin-bottom: 40px;
+
+    @include Tablet {
+      margin-bottom: 100px;
+    }
+  }
+
+  &__list {
+    text-align: center;
+    list-style: none;
+    text-decoration: none;
+    font-weight: 600;
+    font-size: 18px;
+    text-transform: capitalize;
+
+    &--item {
+      border-bottom: 1px solid $border-links-color;
+      margin-bottom: 30px;
+      padding-bottom: 10px;
+
+      @include Tablet {
+        margin-bottom: 50px;
+      }
+    }
+
+    &--link {
```

Comment:
Use '__link' instead of '--link' for BEM element names. '--' is used for modifiers.